src/main/java/network/brightspots/rcv/StreamingCVRReader.java

```diff
@@ -338,6 +356,15 @@ public void headerFooter(String s, boolean b, String s1) {
     if (this.unrecognizedCandidateCounts.size() > 0) {
```

Comment:
you don't need `this.`

...unless this is a deliberate stylistic thing? But we don't do it elsewhere, AFAIK.

Reply:
Oh that's just a habit from writing old c++